### PR TITLE
translate v6.0 documents into Chinese

### PR DESCRIPTION
**Pull request by labilezhu**
translate v6.0 documents into Chinese. Includeing User Documents and README.md  of project.

translate method ref: http://www.sphinx-doc.org/en/stable/intl.html

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -167,5 +167,6 @@
      'Miscellaneous'),
 ]
 
-
-
+language = 'es' # language supported
```
Comment (bbakerman) [inline docs/conf.py]: es is Spanish not english.  You want 'en'

Comment (labilezhu) [reply docs/conf.py]: Ok, I have fixed it.

**Comment by labilezhu** [comment]
Ok, I have fixed it.

**Comment by bbakerman** [comment]
Thanks for much for doing this PR.  For Chinese readers it helps spread the knowledge.

I do worry that this will quickly get out of date, since the .po keys are text from the source document.  I guess that is how .po files work (I have not worked with them in anger) and so be it.

We will be aggressively adding new documentation as the project progresses.

I worry that this will atrophy but some documentation is better than none.

**Comment by labilezhu** [comment]
To help anyone to make the zh_cn translate easy, and keep updated.  I commit my OmegaT 3.6.0 (An open source CAT tool for translation) project to [graphql-java-i18n-zh_cn](https://github.com/labilezhu/graphql-java-i18n-zh_cn)  :

It helps anyone to translate PO files. Any one want to translating graphql-java documents into Chinese can use this OmegaT project.

This will save your time for catch up the continuous updated English doucment. Because OmegaT can remember the translated result by sentence, and reuse it when the source English document updated.

**Pull request merged**